dashboard/src/components/Config/PkgRepoList/PkgRepoList.tsx

```diff
@@ -172,6 +196,7 @@ function PkgRepoList() {
             namespace={currentNamespace}
             helmGlobalNamespace={helmGlobalNamespace}
             carvelGlobalNamespace={carvelGlobalNamespace}
+            disabled={!supportedCluster}
```

Comment:
Also disabled the "Add package repository" button if the cluster is not the default one.